requirements-full.txt

```diff
@@ -84,6 +85,7 @@ jupyter-core==4.7.1; python_full_version >= "3.6.1" and python_version >= "3.6"
 jupyter==1.0.0
 jupyterlab-pygments==0.1.2; python_version >= "3.6"
 jupyterlab-widgets==1.0.0; python_version >= "3.6"
+keras-nightly==2.5.0.dev2021032900
```

Comment:
Still not sure what this is.  2.5 matches TF?

Reply:
I've just removed it, will merge once checks go through